envs/runners/adversarial_runner.py

```diff
@@ -789,13 +934,22 @@ def run(self):
             if self.base_levels == 'batch':
                 fixed_seeds = env_info
             elif self.base_levels == 'easy':
+                if self.use_accel_paired:
+                    # paired signed regret score
+                    regret_score = self._calculate_paired_regret_scores(agent_info, adversary_agent_info, type=args.accel_paired_score_function)
                 if args.num_processes >= 4:
                     # take top 4
-                    easy = list(np.argsort((agent_info['mean_return'].detach().cpu().numpy() - agent_info['batched_value_loss'].detach().cpu().numpy()).flatten())[:4])
+                    if self.use_accel_paired:
+                        easy = list(np.argsort((regret_score.detach().cpu().numpy()).flatten())[:4])
```

Comment:
Do I need to flatten or not? My old results are on un-flattend array and hence only single level was mutated as pointed out by this issue #11